README.adoc

```diff
@@ -163,6 +163,7 @@ spring.cloud.vault:
     scheme: http
     connection-timeout: 5000
     read-timeout: 15000
+    token: some-application-token
```

Comment:
Right, this document does not mention authentication at all. I created a new ticket to improve the quickstart guide. We have docs on authentication in [spring-cloud-vault-config.adoc](https://github.com/spring-cloud-incubator/spring-cloud-vault-config/blob/master/docs/src/main/asciidoc/spring-cloud-vault-config.adoc#authentication-methods)